src/sentry/tasks/reports.py

```diff
@@ -749,7 +749,7 @@ class DistributionType(NamedTuple):
 
 def build_project_breakdown_series(reports):
     def get_legend_data(report):
-        accepted_errors, accepted_transactions, filtered, rate_limited = report.series_outcomes
+        accepted_transactions, accepted_errors, filtered, rate_limited = report.series_outcomes
```

Comment:
Having the output be a dict (comes from here right? https://github.com/getsentry/sentry/blob/037059e2c810423ddd5de92c0a12010ec1cc8454/src/sentry/tasks/reports.py#L357) might be nicer so we never hit this again, but 🤷